app/models/form/sales/questions/other_buyer2_relationship_to_buyer1.rb

```diff
@@ -0,0 +1,10 @@
+class Form::Sales::Questions::OtherBuyer2RelationshipToBuyer1 < ::Form::Question
+  def initialize(id, hsh, page)
+    super
+    @id = "otherrelat2"
```

Comment:
Is this the question that should be hidden in check answers? (i.e. if you haven't said the relationship is other, we don't need to show "what other"?

Reply:
Yep, I think I'm doing that already - here's what the CYA page looks like in the two cases:
Non-other:
<img width="896" alt="image" src="https://user-images.githubusercontent.com/94526761/193231641-d63bb830-f404-454c-9952-1ae1fd4a38d7.png">
Other with user input:
<img width="862" alt="image" src="https://user-images.githubusercontent.com/94526761/193231592-5e7baa29-77c8-454c-8598-3dcb4876e550.png">